pokemongo_bot/cell_workers/move_to_map_pokemon.py

```diff
@@ -249,8 +249,14 @@ def work(self):
 
         pokemon = pokemon_list[0]
 
+
+        min_ball = 1
+        #Load the min amount of ball from configuration file
+        if self.config['min_ball']:
```

Comment:
Can you move this into the initialize function? 

`self.min_ball = self.config.get('min_ball', 1)`